docs/user/labels.md

```diff
@@ -147,6 +147,56 @@ deployment process.
 
    Default value: `true`
 
+## OIDC
+
+* `oidc_enabled`
+
+   Enable OpenID Connect for the kube-api login.
+
+   Default value: `false`
```

Comment:
This works great, but I have some ideas around cleaning up the UI:

Since `oidc_issuer_url` is an absolute requirement, no matter what, it will always be set if you're using OpenID connect, we can probably use that as a label to decide if it's on or not (and not rely on `oidc_enabled` instead).